english/virtualreality/oculus.md

```diff
@@ -3,7 +3,7 @@ feeling when you first got your Oculus Rift, you must have fallen asleep with it
 while playing Marshmallow Kingdom again!
 
 You take it off and return to the real world.
-Checkout Samsung VR for amazing experiance.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.